insight-rest-objects/src/main/java/org/nuxeo/ai/sdk/objects/TensorInstances.java

```diff
@@ -32,6 +32,27 @@
 
 /**
  * A JSON representation of Tensorflow instance parameters
+ *
+ * <code>
+ * {
+ *   "docId": "aDocumentId",
+ *   "instances": [
+ *     {
+ *       "dc:subjects": {
+ *         "type": "cat",
+ *         "categories": [
+ *           "sciences",
+ *           "art/cinema"
+ *         ]
+ *       },
+ *       "dc:title": {
+ *         "type": "txt",
+ *         "text": "My document title"
+ *       }
+ *     }
+ *   ]
+ * }
```

Comment:
Should have a representation of all types. Missing image.

Reply:
It's an example of output not documentation of API definition.

Reply:
This is the documentation. We have no other place with documentation.

Reply:
don't confuse brief java doc and a tutorial. The documentation doesn't provide exhaustive json definitions for all the scenarios.